app/javascript/packs/components/browser-detection.js

```diff
@@ -0,0 +1,19 @@
+import Bowser from "bowser";
+const browser = Bowser.getParser(window.navigator.userAgent);
+
+document.addEventListener('turbolinks:load', function() {
+  const isInvalidValidBrowser = browser.satisfies({
```

Comment:
is invalid valid, je suis perdu :)
propositions pour le nom de la variable : browserIsTooOld ? oldBrowser ?